packages/lesswrong/server/repos/VotesRepo.ts

```diff
@@ -496,7 +496,27 @@ class VotesRepo extends AbstractRepo<"Votes"> {
     );
     return results.map(({ vote_id }) => vote_id);
   }
-
+  
+  async getVotesOnSamePost({userId, postId, excludedDocumentId}: {
+    userId: string,
+    postId: string,
+    excludedDocumentId: string,
+  }): Promise<DbVote[]> {
+    return await this.manyOrNone(`
+      SELECT
+        v.*
+      FROM
+        "Votes" v
+        "Comments" c
```

Comment:
Missing a join?

Reply:
This might be clearer written as a `LEFT JOIN` rather than as two tables in the `FROM` clause, but I believe it does work (the join constraint is `c._id = v."documentId"` and is in the `WHERE` clause). (Note that this is shown in the Github PR with a missing comma; that was fixed in the next commit.)